### PR TITLE
Add several containers, Sorting category and timsort, Nanotimer to benchmarking

### PR DESCRIPTION
**Pull request by mattreecebentley**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -224,11 +224,16 @@ A curated list of awesome C++ (or C) frameworks, libraries, resources, and shiny
...
 * [Hopscotch map](https://github.com/Tessil/hopscotch-map) - A fast header-only hash map which uses hopscotch hashing for collisions resolution. [MIT]
 * [LSHBOX](https://github.com/RSIA-LIESMARS-WHU/LSHBOX) - A c++ toolbox of locality-sensitive hashing (LSH), provides several popular LSH algorithms, also support Python and MATLAB. [GPL]
+* [plf::list](https://github.com/mattreecebentley/plf_list) - A std::list implementation which removes range splicing in order to enable cache-friendlier structure, yielding significant performance gains. [zLib] [website](http://www.plflib.org/list.htm)
+* [plf::stack](https://github.com/mattreecebentley/plf_stack) - A replacement container for the std::stack container adaptor, with better performance than any std container in a stack context. [zLib] [website](http://www.plflib.org/stack.htm)
+* [ring_span lite](https://github.com/martinmoene/ring-span-lite) - A simplified implementation of Guy Davidson & Arthur O'dwyer's ring_span - a circular buffer view. [MIT]
 * [sparsepp](https://github.com/greg7mdp/sparsepp) - A fast, memory efficient hash map for C++. [BSD 3-clause]
 
+
```
Comment (fffaraz) [inline README.md]: Please remove this empty line.

File: README.md [hunk 2]
```diff
@@ -602,6 +608,10 @@ A curated list of awesome C++ (or C) frameworks, libraries, resources, and shiny
 * [protobuf-c](https://github.com/protobuf-c/protobuf-c) - Protocol Buffers implementation in C. [BSD]
 * [SimpleBinaryEncoding](https://github.com/real-logic/simple-binary-encoding) - encoding and decoding application messages in binary format for low-latency applications. [Apache2]
 
+## Sorting
```
Comment (fffaraz) [inline README.md]: Please add a link to the new category in the index section.

**Comment by mattreecebentley** [comment]
Changes implemented.

File: README.md (hunk 1 above)
Comment (mattreecebentley) [reply README.md]: Done

File: README.md (hunk 2 above)
Comment (mattreecebentley) [reply README.md]: Done

**Pull request merged**